src/domain/ruleset-repository.ts

```diff
@@ -250,6 +250,10 @@ export class RulesetRepository extends Repository {
   public get config(): Configuration {
     return this._config;
   }
+
+  public getAllMaintainers(): ReadonlyArray<Maintainer> {
+    return  Object.values(Object.values(this._metadataTemplate).flatMap(template => template.maintainers).reduce((maintainers, curr) => ({...maintainers, [curr.email]: curr}), {}));
```

Comment:
rando nit: I noticed we don't have any formatter setup here

Reply:
I just forgot to install the pre-commit on the machine I'm using. Updated with formatting.